plugins/vpp/ifplugin/interface_state.go

```diff
@@ -164,23 +156,10 @@ func (c *InterfaceStateUpdater) subscribeVPPNotifications() error {
 		return errors.Errorf("%s returned %d", wantIfEventsReply.GetMessageName(), wantIfEventsReply.Retval)
 	}
 
-	wantStatsReply := &stats.WantStatsReply{}
-	// enable interface counters notifications from VPP
-	err = c.vppCh.SendRequest(&stats.WantStats{
-		PID:           uint32(os.Getpid()),
-		EnableDisable: 1,
-	}).ReceiveReply(wantStatsReply)
-	if err != nil {
-		return errors.Errorf("failed to get interface events: %v", err)
-	}
-	if wantStatsReply.Retval != 0 {
-		return errors.Errorf("%s returned %d", wantStatsReply.GetMessageName(), wantStatsReply.Retval)
-	}
-
 	return nil
 }
 
-// Close unsubscribes from interface state notifications from VPP & GOVPP channel
+// Close un-subscribes from interface state notifications from VPP & GOVPP channel
```

Comment:
Why the change? The word [_unsubscribe_ is correct](https://www.dictionary.com/browse/unsubscribe).

Reply:
according to the default GoLand dictionary, it's a typo.. so let's save it manually

Reply:
Weird.. for this reason I turned of spell check inspection in GoLand, heh.